### DISCUSSION TITLE
OSC type = none (or off)

### DISCUSSION THREAD

**Discussion by enorrmann** [bangcorrupt/hacktribe#320] [Ideas]
When you assign a sample to a pattern and then delete that sample, the osc type is set to a blank name, and  I found that usefull to sequence external gear without consuming processing power to play that osc, even when the track is not muted.
But when rotating the sample dial, there is no way to select a "none" or "off" oscilator. The closest would be to use a blank sample or audio input mono but even that I think that track uses dsp power to fx and filters.
It would be nice to be able to select a "off" style sample using the sample wheel.
What do you think ?

**Comment by bangcorrupt** [comment]
Nice catch, I didn't know we could do that by deleting the sample.  I'll try to look at what happens when the oscillator is set to a sample that doesn't exist.  Do you know if this reduces voice stealing compared to using the Audio In oscillator, as mentioned in #177?

**Comment by enorrmann** [comment]
sadly I did some more testing and it seems the voce stealing is present also with this type of osc
